### PR TITLE
Fix: app crash at NewMessagesMatcher

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

App crashes ar `Wire.NewMessagesMatcher.description()`

### Causes

One of the localization is in the wrong format

### Solutions

Fix the localization formatter.
Created a test to loop through all supported languages and case for message status.

### TODO:
- [x] import the localization resource again

### CODE REVIEW

File: Wire-iOS/Resources/zh-Hans.lproj/Localizable.stringsdict [hunk 1]
```diff
@@ -1,324 +1,323 @@
...
+		<key>NSStringLocalizedFormatKey</key>
+		<string>%#@d_number_of_new@</string>
+		<key>d_number_of_new</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>d</string>
+			<key>other</key>
+			<string>%d条消息</string>
```
Comment (billypchan) [inline Wire-iOS/Resources/zh-Hans.lproj/Localizable.stringsdict]: cause of the crash

File: Wire-iOS/Sources/UserInterface/ConversationList/ZMConversation+Status.swift [hunk 2]
```diff
@@ -71,6 +71,34 @@ enum StatusMessageType: Int {
     case addParticipants
     case removeParticipants
     case newConversation
+
+    private var localizationSilencedRootPath: String {
+        return "conversation.silenced.status.message"
+    }
+
+    private var matchedSummaryTypesDescriptions: [StatusMessageType: String] { ///TODO: switch
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/ConversationList/ZMConversation+Status.swift]: Todo?

File: Wire-iOS Tests/Status/StatusMessageTypeTests.swift [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+final class StatusMessageTypeTests: XCTestCase {
+    var sut: StatusMessageType!
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
```
Comment (johnxnguyen) [inline Wire-iOS Tests/Status/StatusMessageTypeTests.swift]: You could delete these.

File: Wire-iOS/Sources/UserInterface/ConversationList/ZMConversation+Status.swift [hunk 4]
```diff
@@ -76,18 +76,19 @@ enum StatusMessageType: Int, CaseIterable {
...
-        .text:       "generic_message"
-        ]
+    private var localizationKeySuffix: String? {
+        switch self {
+            case .mention: return    "mention"
+            case .reply: return      "reply"
+            case .missedCall: return "missedcall"
+            case .knock: return      "knock"
+            case .text: return       "generic_message"
+            default: return          nil
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/ConversationList/ZMConversation+Status.swift]: This is a bit messy, could you clean this up?

**Pull request merged**